source/connectors/kafka-sink-cdc.txt

```diff
@@ -0,0 +1,78 @@
+.kafka-sink-cdc
+
+==============================
+Kafka Sink Change Data Capture
+==============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecols
+
+Change Data Capture
+-------------------
+
+
+Change Data Capture Mode
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+The sink connector can also be used in a different operation mode in order to handle change data capture (CDC) events.
+Currently, the following CDC events from `Debezium <http://debezium.io/>`_ can be processed:
+
+
+* `MongoDB <http://debezium.io/docs/connectors/mongodb/>`_
+* `MySQL <http://debezium.io/docs/connectors/mysql/>`_
+* `PostgreSQL <http://debezium.io/docs/connectors/postgresql/>`_
+
+This effectively allows to replicate all state changes within the source databases into MongoDB collections. Debezium produces very similar
+CDC events for MySQL and PostgreSQL. The so far addressed use cases worked fine based on the same code which is why there is only one
+*RdbmsHandler* implementation to support them both at the moment.
+
+Also note that **both serialization formats (JSON+Schema & AVRO) can be used** depending on which configuration is a better fits for your
+use case.
+
+CDC Handler Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The sink connector configuration offers a property called *mongodb.change.data.capture.handler* which is set to the fully qualified class
+name of the respective CDC format handler class. These classes must extend from the provided abstract class
+`CdcHandler <https://github.com/mongodb/mongo-kafka/blob/master/src/main/java/com/mongodb/kafka/connect/sink/cdc/CdcHandler.java>`_.
+As soon as this configuration property is set the connector runs in **CDC operation mode**.
+
+An example JSON configuration the sink connector which uses the current default implementation that is capable to process
```

Comment:
configuration *is*?

Reply:
connector which -> connector, which

https://www.grammarly.com/blog/comma-before-which/